cmd/console/console.go

```diff
@@ -210,9 +210,10 @@ func modifySshdConfig() error {
 	return ioutil.WriteFile("/etc/ssh/sshd_config", []byte(sshdConfigString), 0644)
 }
 
+// https://www.freedesktop.org/software/systemd/man/os-release.html
 func writeOsRelease() error {
 	idLike := "busybox"
-	if osRelease, err := ioutil.ReadFile("/etc/os-release"); err == nil {
+	if osRelease, err := ioutil.ReadFile("/usr/lib/os-release"); err == nil {
```

Comment:
I picked this direction due to 

```
/etc/os-release should be a relative symlink to /usr/lib/os-release, to provide compatibility with applications only looking at /etc.
```

So - I guess its time for some larger context - why are we writing this file dynamically using `ros`?